packages/schema/parse/src/typeInfo/enum.ts

```diff
@@ -0,0 +1,5 @@
+export function isEnumType(type: string, enumTypes: string[]): boolean {
```

Comment:
Ideally we can get rid of this, not sure it's needed if we can solve the dependency talked about above.